src/stores/ApiRequestStores/SoftApApiStores.js

```diff
@@ -0,0 +1,19 @@
+// @flow
+
+import type { WifiNetwork } from '../../types';
+
```

Comment:
I'm worried about using a store to cache this stuff. 

If I run through WiFi setup twice I want to be sure that all these caches are flushed.

Reply:
I'll add disposing logic somewhere, not sure where exactly yet though.